README.md

```diff
@@ -77,6 +77,8 @@ types of algorithms, depending on your use case:
 - The `Transformer` is the more general one (both the `Consumer` and the
   `Producer` are a particular case of this one) and takes both inputs and
   outputs
+- The `Filter`, for removing events from the events that are going to be
```

Comment:
```suggestion
- The `FilterPredicate`, for removing events from the events that are going to be
```

Reply:
I think this explanation is a bit misleading since, the even will be marked to be filtered out only in the current branch of the workflow (and whether it's dropped there depends on the properties of the Sequencer/DecisionHub to which it belongs)

How about something like this:
for determining/controling whether subsequent algorithms in the sequence are executed or bypassed